docs/csharp/language-reference/keywords/out-parameter-modifier.md

```diff
@@ -43,14 +43,12 @@ You can't use the `in`, `ref`, and `out` keywords for the following kinds of met
   
 -   Iterator methods, which include a [yield return](../../../csharp/language-reference/keywords/yield.md) or `yield break` statement.  
 
-## Declaring `out` arguments   
+## Declaring `out` parameters   
 
- Declaring a method with `out` arguments is useful when you want a method to return multiple values. The following example uses `out` to return three variables with a single method call. Note that the third argument is assigned to null. This enables methods to return values optionally.  
+Declaring a method with `out` arguments is the classic way to return multiple values. Beginning with C# 7.0 [tuples](../../tuples.md) are preferred. The following example uses `out` to return three variables with a single method call. Note that the third argument is assigned to null. This enables methods to return values optionally.  
```

Comment:
I think that "tuples are preferred" needs some qualification:
- tuples are most useful for internal methods
- the Try pattern, which returns a Boolean and includes an out argument, is preferable to a tuple.
```suggestion
Declaring a method with `out` arguments is a classic workaround to return multiple values. Beginning with C# 7.0 [tuples](../../tuples.md) are preferred. The following example uses `out` to return three variables with a single method call. Note that the third argument is assigned to null. This enables methods to return values optionally.  
```